core/modules/replica/ConfigurationMySQL.h

```diff
@@ -69,34 +86,274 @@ class ConfigurationMySQL
      */
     ConfigurationMySQL(database::mysql::ConnectionParams const& connectionParams);
 
-    ~ConfigurationMySQL() override = default;
+    ~ConfigurationMySQL() final = default;
+
+    /**
+     * @see Configuration::prefix()
+     */
+    virtual std::string prefix() const final;
```

Comment:
In this design, further inheritance makes no sense. So, I chose to "put a lid" on it here. Everywhere I'm using `final` I'm doing this on purpose, as a safety measure, not just because I like this keyword.